bioconda_utils/cli.py

```diff
@@ -497,18 +526,18 @@ def dag(recipe_folder, config, packages="*", format='gml', hide_singletons=False
      help="""Bump package build numbers even if the only applicable pinning
      change is the python version. This is generally required unless you plan
      on building everything.""")
-def update_pinning(recipe_folder, config, packages="*", skip_if_in_channels=['bioconda', 'bioconda/label/gcc7'], bump_only_python=False):
-    """
-    Bump a package build number and all dependencies as required due to a change in pinnings
+@enable_logging()
+def update_pinning(recipe_folder, config, packages="*",
+                   skip_if_in_channels=['bioconda', 'bioconda/label/gcc7'],
```

Comment:
Can you remove `bioconda/label/gcc7`? We can now exclude that from everywhere in the code.

Reply:
Removed both, actually. You can now only add to the channel list. We don't need to bump stuff that's got the right build number in another channel.